lib/linguist/heuristics.yml

```diff
@@ -218,6 +218,10 @@ disambiguations:
     pattern: '^public\s+(?:SharedPlugin(?:\s+|:)__pl_\w+\s*=(?:\s*{)?|(?:void\s+)?__pl_\w+_SetNTVOptional\(\)(?:\s*{)?)'
   - language: POV-Ray SDL
     pattern: '^\s*#(declare|local|macro|while)\s'
+- extensions: ['.iuml']
+  rules:
+  - language: PlantUML
+    named_pattern: plantuml
```

Comment:
Heuristic rules are used to disambiguate between several Linguist languages using the same file extension. It won't help here.

Reply:
Added another .iuml example in 274ac5a38

Reply:
Got rid of heuristics changes in ee2a91351a